source/process/documentation-UItext-guidelines.rst

```diff
@@ -0,0 +1,193 @@
+User Interface Text Guidelines
+===============================
+This is the Mattermost style guide for text used in the Mattermost product. Use these guidelines to ensure the the text on the Mattermost user interface (UI) is consistent and clear.  These guidelines cover two types of UI text:
+
+- The messages that are triggered because something happened
+- The static text on the user interface
+
+.. note::
+
+Contributions that do not adhere to this style guide will receive edit requests; we'll merge your contribution once you edit the text to conform to these guidelines. You might want to look at the `UX design guidelines`_ as well.
+
+....
+
+.. contents:: 
+   :depth: 2
+
+....
+
+General principles for in-product text
+---------------------------------------
+
+- Keep in mind that your users are from all over the world. Use words that are clear and easy to understand.
+- Don't blame the user. Inform them about what happened, explain why it happened, and suggest a way forward.
```

Comment:
This seems like it should live under the error messages section?

Reply:
Moved it to the messages section.